src/util/event_loop_signal.hpp

```diff
@@ -21,11 +21,11 @@
 
 #include <realm/util/features.h>
 
-#if (defined(REALM_HAVE_UV) && REALM_HAVE_UV && !REALM_PLATFORM_APPLE) || REALM_PLATFORM_NODE
+#if (defined(REALM_HAVE_UV) && REALM_HAVE_UV && !REALM_PLATFORM_APPLE) || (defined(REALM_PLATFORM_NODE) && REALM_PLATFORM_NODE)
 #define REALM_USE_UV 1
 #endif
 
-#if REALM_USE_UV
+#if defined(REALM_USE_UV) && REALM_USE_UV
```

Comment:
I did consider that. I'll do that. Just to be clear, the current approach for `REALM_PLATFORM_NODE` is fine because it's defined in CMake, right?